infra/docker/jobservice/Dockerfile

```diff
@@ -27,4 +27,4 @@ RUN wget -q https://github.com/grpc-ecosystem/grpc-health-probe/releases/downloa
          -O /usr/bin/grpc-health-probe && \
     chmod +x /usr/bin/grpc-health-probe
 
-CMD ["feast", "server"]
+CMD ["python", "-m", "feast.cli", "server"]
```

Comment:
Any reason for this change?

Reply:
container was failing with 
```
pkg_resources.DistributionNotFound: The 'Click==7.*' distribution was not found and is required by feast
```
It could be due to any reason, since it's conda.